jsre/ethereum_js.go

```diff
@@ -5562,10 +5562,18 @@ var methods = function () {
         inputFormatter: [formatters.inputAddressFormatter]
     });
 
+    var importAccount = new Method({
+        name: 'importAccount',
+        call: 'personal_importAccount',
+        params: 2,
+        inputFormatter: [null, null]
+    });
```

Comment:
Yes, it belongs in "personal". ethereum_js.go contains the unmodified source of web3.js, though.
If you make changes here, they will be probably be overwritten when we pull it into geth from upstream. This is why package web3ext exist. Please add an entry for this extension there instead.